openhtf/io/user_input.py

```diff
@@ -37,7 +37,6 @@
                       300,
                       'User prompt timeout in seconds.')
 
-
```

Comment:
2 blank lines between top-level stuff is conventional. I would leave this line in.